doc/commands/install.8.rst

```diff
@@ -25,15 +25,23 @@
 Synopsis
 ========
 
-``dnf5 install [options] <package-spec>...``
+``dnf5 install [options] <package-spec>|@<group-spec>|@<environment-spec>...``
 
 
 Description
 ===========
 
-The ``install`` command in ``DNF5`` is used for installing packages. It makes sure that
-all given packages defined in ``package-spec`` arguments and their dependencies are installed
-on the system. Environments can be installed with ``@environment-id`` as ``<package-spec>``.
+The ``install`` command in ``DNF5`` is used for installing packages, groups or environments.
+
+When installing packages defined in ``package-spec`` arguments, ``DNF5`` ensures that the packages
+and their dependencies are installed on the system.
```

Comment:
> ensures that the packages and their dependencies are installed on the system.

Hmm, it seems we are still promising to do something that we aren't actually doing, e.g. what's described in https://github.com/rpm-software-management/dnf5/issues/1498 ...

Reply:
Consider adding something like: "If a specified package is already installed, DNF5 does not check its dependencies again and simply verifies that the package itself is present."

Reply:
If the described situation also occurs for group and env specs, I’d suggest adding it as a general note under the first sentence of the description.

Reply:
I tried to phrase it in such a way that specifies we check the dependencies only when actually installing the package so I think it should cover the mentioned issue.

However I can add the sentence to make it explicit.

Reply:
> If the described situation also occurs for group and env specs, I’d suggest adding it as a general note under the first sentence of the description.

It doesn't, both groups and environments can actually be installed (synchronized) over and over again and each time the dependencies/content is processed.

I have added the note specific for packages.

Reply:
> However I can add the sentence to make it explicit.

Thanks, it's now more understandable and explicit.